DFC.App.DiscoverSkillsCareers.Core/Constants/RouteName.cs

```diff
@@ -2,6 +2,8 @@
 {
     public static class RouteName
     {
-        public const string Prefix = "discover-your-skills-careers";
+        public const string Prefix = "discover-your-skills-and-careers";
+
+        public const string ReturnLink = "/discover-your-skills-and-careers";
```

Comment:
You could concatenate Prefix here instead of repeating "discover-your-skills-and-careers"